### PR TITLE
Fixed instances of e.g. and i.e.

### PR DESCRIPTION
**Pull request by tompratt-AQ**
## Summary
Replaced all instances of e.g. and i.e. in .md files with "for example," and "that is,".
Insert a short (one or two sentence) summary here.

## Suggested Reviewers
@mairaw

### CODE REVIEW

File: docs/csharp/programming-guide/concepts/linq/how-to-use-annotations-to-transform-linq-to-xml-trees-in-an-xslt-style.md [hunk 1]
```diff
@@ -31,7 +31,7 @@ Annotations can be used to facilitate transforms of an XML tree.
 <text>A phrase with <b>bold</b> and <i>italic</i> text.</text>  
 ```  
   
- For any given text node, there may be any number of child `<b>` and `<i>` elements. This approach extends to a number of other situations: i.e. pages that can contain a variety of child elements, such as regular paragraphs, bulleted paragraphs, and bitmaps. Cells in a table may contain text, drop down lists, or bitmaps. One of the primary characteristics of document centric XML is that you do not know which child element any particular element will have.  
+ For any given text node, there may be any number of child `<b>` and `<i>` elements. This approach extends to a number of other situations: that is, pages that can contain a variety of child elements, such as regular paragraphs, bulleted paragraphs, and bitmaps. Cells in a table may contain text, drop down lists, or bitmaps. One of the primary characteristics of document centric XML is that you do not know which child element any particular element will have.
```
Comment (rpetrusha) [inline docs/csharp/programming-guide/concepts/linq/how-to-use-annotations-to-transform-linq-to-xml-trees-in-an-xslt-style.md]: the i.e. wasn't optimal here, so replacing it only slightly helps. Possibly better is "...other situations, such as pages that can contain a variety of child elements like regular paragraphs, bulleted paragraphs, and bitmaps"

Also, document-centric.

File: docs/csharp/programming-guide/concepts/object-oriented-programming.md [hunk 2]
```diff
@@ -314,7 +314,7 @@ var sampleObject =
  Inheritance enables you to create a new class that reuses, extends, and modifies the behavior that is defined in another class. The class whose members are inherited is called the *base class*, and the class that inherits those members is called the *derived class*. However, all classes in C# implicitly inherit from the <xref:System.Object> class that supports .NET class hierarchy and provides low-level services to all classes.  
   
 > [!NOTE]
->  C# doesn't support multiple inheritance, i.e. you can specify only one base class for a derived class.  
+>  C# doesn't support multiple inheritance, that is, you can specify only one base class for a derived class.
```
Comment (rpetrusha) [inline docs/csharp/programming-guide/concepts/object-oriented-programming.md]: i.e. again was awkward. I'd make these two sentences.

File: docs/standard/base-types/formatting-types.md [hunk 3]
```diff
@@ -1180,7 +1180,7 @@ Console.WriteLine(result)
...
 
-* The specific way in which an object is represented as a string, if the object implements the [IFormattable](xref:System.IFormattable) interface and supports format strings. You do this by following the format item's index with a : (colon) followed by a valid format string. The previous example did this by formatting a date value with the "d" (short date pattern) format string (e.g., `{0:d}`) and by formatting a numeric value with the "C2" format string (e.g., `{2:C2}` to represent the number as a currency value with two fractional decimal digits. 
+* The specific way in which an object is represented as a string, if the object implements the [IFormattable](xref:System.IFormattable) interface and supports format strings. You do this by following the format item's index with a : (colon) followed by a valid format string. The previous example did this by formatting a date value with the "d" (short date pattern) format string (for example, `{0:d}`) and by formatting a numeric value with the "C2" format string (for example, `{2:C2}` to represent the number as a currency value with two fractional decimal digits.
```
Comment (rpetrusha) [inline docs/standard/base-types/formatting-types.md]: This also needs a closing paren -- ...digits).

File: docs/standard/common-type-system.md [hunk 4]
```diff
@@ -25,7 +25,7 @@ In order to do this transparently, there has to be a common way to describe all
 *   Establish a framework for cross-language execution.
 *   Provide an object-oriented model to support implementing various languages on .NET platform.
 *   Define a set of rules that all languages must follow when it comes to working with types.
-*   Provide a library that contains the basic primitive types that are used in application development (i.e. `Boolean`, `Byte`, `Char` etc.)
+*   Provide a library that contains the basic primitive types that are used in application development (that is, `Boolean`, `Byte`, `Char` etc.)
```
Comment (rpetrusha) [inline docs/standard/common-type-system.md]: "such as" reads somewhat better than "that is" here.

File: docs/visual-basic/programming-guide/concepts/linq/how-to-use-annotation-trees-to-transform-linq-to-xml-trees-in-an-xslt-style.md [hunk 5]
```diff
@@ -31,7 +31,7 @@ Annotations can be used to facilitate transforms of an XML tree.
 <text>A phrase with <b>bold</b> and <i>italic</i> text.</text>  
 ```  
   
- For any given text node, there may be any number of child `<b>` and `<i>` elements. This approach extends to a number of other situations: i.e. pages that can contain a variety of child elements, such as regular paragraphs, bulleted paragraphs, and bitmaps. Cells in a table may contain text, drop down lists, or bitmaps. One of the primary characteristics of document centric XML is that you do not know which child element any particular element will have.  
+ For any given text node, there may be any number of child `<b>` and `<i>` elements. This approach extends to a number of other situations: that is, pages that can contain a variety of child elements, such as regular paragraphs, bulleted paragraphs, and bitmaps. Cells in a table may contain text, drop down lists, or bitmaps. One of the primary characteristics of document centric XML is that you do not know which child element any particular element will have.
```
Comment (rpetrusha) [inline docs/visual-basic/programming-guide/concepts/linq/how-to-use-annotation-trees-to-transform-linq-to-xml-trees-in-an-xslt-style.md]: See earlier comment.

File: docs/visual-basic/programming-guide/concepts/object-oriented-programming.md [hunk 6]
```diff
@@ -325,7 +325,7 @@ Dim sampleObject =
  Inheritance enables you to create a new class that reuses, extends, and modifies the behavior that is defined in another class. The class whose members are inherited is called the *base class*, and the class that inherits those members is called the *derived class*. However, all classes in Visual Basic implicitly inherit from the <xref:System.Object> class that supports .NET class hierarchy and provides low-level services to all classes.  
   
 > [!NOTE]
->  Visual Basic doesn't support multiple inheritance, i.e. you can specify only one base class for a derived class.  
+>  Visual Basic doesn't support multiple inheritance, that is, you can specify only one base class for a derived class.
```
Comment (rpetrusha) [inline docs/visual-basic/programming-guide/concepts/object-oriented-programming.md]: See earlier comment.

**Review by rpetrusha — CHANGES_REQUESTED** [review]

**Comment by tompratt-AQ** [comment]
Addressed the review comments and took all of the suggestions from rpetrusha.

**Review by cartermp — APPROVED** [review]

**Review by rpetrusha — APPROVED** [review]
LGTM.

**Pull request merged**